composer.json

```
@@ -27,7 +27,7 @@
     "behat/mink-extension": "~2.0",
     "behat/mink-goutte-driver": "~1.0",
     "behat/mink-selenium2-driver": "~1.1",
-    "drupal/drupal-driver": "^2.0.0",
+    "drupal/drupal-driver": "^2.0.0-alpha5",
```

This should go back to `^2.0.0` I think?
```suggestion
    "drupal/drupal-driver": "^2.0.0",
```

---

To `^2.1.0` now to match `master`.